mime.go

```diff
@@ -47,6 +47,19 @@ func (m *MailYak) buildMimeWithBoundaries(w io.Writer, mb, ab string) error {
 		return err
 	}
 
+	var (
+		hasBody        = m.html.Len() != 0 || m.plain.Len() != 0
+		hasAttachments = len(m.attachments) != 0
+	)
+
+	// if we don't have text/html body or attachments we can skip Content-Type header
+	// in that case next default will be assumed
+	//    Content-type: text/plain; charset=us-ascii
+	// See https://datatracker.ietf.org/doc/html/rfc2045#page-14
+	if !hasBody && !hasAttachments {
```

Comment:
I really like how you expressed this - the code is nice! 

And thank you for a comment describing why _and linking to the relevant part of the RFC_ :heart:

Reply:
Thank you!